Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM jinaai/jina:2.0.0rc6 as BASE
+
+# install git
+RUN apt-get -y update && apt-get install -y git
+
+# setup the workspace
+COPY . /workspace
+WORKDIR /workspace
+
+# install the third-party requirements
+RUN pip install -r requirements.txt
+RUN python -c "import clip;clip.load('ViT-B/32', 'cpu', True)"
+
+# for testing the image
+FROM BASE
+RUN pip install -r tests/requirements.txt
+RUN pytest -s -vv
+
+FROM BASE
+ENTRYPOINT ["jina", "pod", "--uses", "config.yml"]
```

Comment:
call "jina executor". We want to hide pod from the user

Reply:
good point